packages/cardano-services/src/WsServer/server.ts

```diff
@@ -220,9 +220,9 @@ export class CardanoWsServer extends WsProvider {
         };
 
         const loadTransactions = async () => {
-          const addressesSet = new Set<Cardano.PaymentAddress>();
-          for (const ws of this.wss.clients) for (const address of ws.addresses) addressesSet.add(address);
-          const addresses = [...addressesSet];
+          const addressesObj: Record<Cardano.PaymentAddress, true> = {};
+          for (const ws of this.wss.clients) for (const address of ws.addresses) addressesObj[address] = true;
```

Comment:
No fancy loops? with those loops performance is more than 3x

At the very least please fix the formatting

Reply:
At the end, the core of the x3 performances is the use of the `Object` rather than `Map` or `Set`.
Formatting is dictated by `es-lint` rules...

Anyway, 9bbae51ef96 (already squashed) should solve this.